caravel/sql_lab.py

```diff
@@ -138,3 +140,9 @@ def get_sql_results(query_id, return_results=True):
         payload['error'] = query.error_message
     if return_results:
         return payload
+    '''
```

Comment:
this part is WiP... should have been in another PR but I was rushing for the beta launch